splice/queries/campaign.py

```diff
@@ -0,0 +1,74 @@
+from sqlalchemy.orm.exc import NoResultFound
+
+from splice.models import Campaign, CampaignCountry
+from splice.queries.common import row_to_dict
+
+
+def get_campaigns(account_id=None):
+    from splice.environment import Environment
+
+    env = Environment.instance()
+
+    query = (
+        env.db.session
+        .query(Campaign, CampaignCountry.country_code)
+        .join(CampaignCountry, Campaign.id == CampaignCountry.campaign_id)
```

Comment:
This join is going to produce multiple rows per campaign, which I don't think is what you want.  Maybe do not return countries in this query, but add another path /campaigns/25/countries